graph-db/serverless/src/IndexCollectionAcl/handler.js

```diff
@@ -0,0 +1,66 @@
+import 'array-foreach-async'
+import gremlin from 'gremlin'
+
+import { getEchoToken } from '../utils/cmr/getEchoToken'
+import { initializeGremlinConnection } from '../utils/gremlin/initializeGremlinConnection'
+import { indexCollectionAcl } from '../utils/cmr/indexCollectionAcl'
+import { deleteCollectionAcl } from '../utils/cmr/deleteCollectionAcl'
+import { fetchAccessControlList } from '../utils/cmr/fetchAccessControlList'
+const gremlinStatistics = gremlin.process.statics //Used for anonymous traversals commonly imported as __
+let gremlinConnection
+let token
+
+const updateActionType = 'concept-update'
+const deleteActionType = 'concept-delete'
+
+const indexAcl = async (event) => {
+    // Prevent creating more tokens than necessary
+  if (token === undefined) {
```

Comment:
Can this token ever not be undefined? If this is executing in a lambda this will always be true.

Reply:
Yea, the first time this lambda fires up, it will be undefined, then it gets set in context and used until it spins down